### PR TITLE
Add the 2201.2.2 release note

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Add the 2201.2.2 release note.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+### Improvements
+
+### Bug Fixes
+
+## Developer tools updates
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake 2201.2.2 of the repositories below.
+
```
Comment (gimantha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:58]:
```suggestion

- [Language](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.2.2+is%3Aclosed)
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    - /downloads/swan-lake-release-notes/2201.2.2/
+    - /downloads/swan-lake-release-notes/2201.2.2-swan-lake/
+    - /downloads/swan-lake-release-notes/2201.2.2-swan-lake
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+
+## Overview of Ballerina Swan Lake 2201.2.2
+
+<em>Swan Lake 2201.2.2 is the second patch release of Ballerina 2201.2.0 (Swan Lake Update 2) and it includes a new set of bug fixes to the language server and developer tooling.</em>
```
Comment (MaryamZi) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:17]: With the change suggested by Gimantha

```suggestion
<em>Swan Lake 2201.2.2 is the second patch release of Ballerina 2201.2.0 (Swan Lake Update 2) and it includes a new set of bug fixes to the language, language server, and developer tooling.</em> 
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md (hunk 1 above)
Comment (MaryamZi) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:58]: Needs to go under the bug fixes section of the language section? L43,

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+### Bug Fixes
+
+## Standard library updates
+
+### New features
+
+### Improvements
+
+### Bug Fixes
```
Comment (MaryamZi) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:51]: Once completed, we can remove the empty sections.

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md (hunk 1 above)
Comment (lasinicl) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:58]: This should come under `## Language updates` similar to what we have done in [2201.1.2 release note](https://ballerina.io/downloads/swan-lake-release-notes/swan-lake-2201.1.2)

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+## Developer tools updates
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake 2201.2.2 of the repositories below.
+
+- [Language Server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FLanguageServer%2FExtensions+milestone%3A2201.2.2+is%3Aclosed)
+- [CLI](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A2201.2.2+label%3AArea%2FCLI)
+- [Project API](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.2.2+is%3Aclosed+label%3AArea%2FProjectAPI)
```
Comment (NipunaRanasinghe) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:61]:
```suggestion
- [Project API](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.2.2+is%3Aclosed+label%3AArea%2FProjectAPI)
- [Debugger](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+milestone%3A2201.2.2+is%3Aclosed+label%3AArea%2FDebugger)
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+2. Run `bal dist update` ( or `bal dist pull 2201.2.2`) to update your Ballerina version to 2201.2.2.
+
+However, if you are using a version below 2201.0.0 (Swan Lake) and if you already ran `bal dist update` (or `bal dist pull 2201.2.2`) before `bal update`, see [Troubleshooting](/downloads/swan-lake-release-notes/2201-0-0-swan-lake/#troubleshooting) to recover your installation.
+
+## Install Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
+
+## Language updates
+
```
Comment (gimantha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:38]:
```suggestion

To view bug fixes, see the [GitHub milestone for Swan Lake 2201.2.2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.2.2+is%3Aclosed)
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md (hunk 1 above)
Comment (gimantha) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:58]: Thanks. my mistake.

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.2.2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.2.2+is%3Aclosed)
```
Comment (gimantha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:39]:
```suggestion
To view bug fixes, see the [GitHub milestone for Swan Lake 2201.2.2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.2.2+is%3Aclosed)

```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+### Bug Fixes
+
+## Standard library updates
+
+### New features
+
+### Improvements
+
+### Bug Fixes
+
```
Comment (lasinicl) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:54]:
```suggestion
````

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+To view bug fixes, see the [GitHub milestone for Swan Lake 2201.2.2](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FCompilerFE+milestone%3A2201.2.2+is%3Aclosed)
+
+## Developer tools updates
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake 2201.2.2 of the repositories below.
+
+- [Language Server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FLanguageServer%2FExtensions+milestone%3A2201.2.2+is%3Aclosed)
```
Comment (lasinicl) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.2-release-note.md:47]:
```suggestion
- [Language Server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3ATeam%2FLanguageServer+milestone%3A2201.2.2+is%3Aclosed)
```

**Review by lasinicl — APPROVED** [review]

**Review by NipunaRanasinghe — APPROVED** [review]

**Review by gimantha — APPROVED** [review]

**Pull request merged**